index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
OK, so names can be confusing. We need to make it very clear which sensor we spec. I actually like the MS names.

* Device Orientation Sensor aka Compass Sensor (potentially with/without gyroscope - if we understand the use-case for that distinction)
* Relative Orientation Sensor aka Game Orientation Sensor

For Web VR use-cases etc we really care about the latter.

I think that we should really keep these separate because the use-cases for them are quite different.